setup.py

```diff
@@ -92,6 +92,7 @@
     'Flask-OAuthlib>=0.6.0,<0.7',  # quick fix for issue invenio#2158
     'Flask-Security>=1.7.5',
     'invenio-accounts>=1.0.0a6',
+    'invenio-db>=1.0.0a9',
```

Comment:
This one is already present in ``extra_requie``. You just have to install with pip install -e .[all,postgresql]